group-aws/bestpractices.md

```diff
@@ -0,0 +1,19 @@
+# Best practices
```

Comment:
This list is very generic, everything should basically have the same list.
So I wonder if this needs to be documented in this space.
A pointer to the generic stuff, and a pointer to the AWS specific stuff seems to suffice.